app/controllers/users_controller.rb

```diff
@@ -16,7 +18,30 @@ def update
     end
   end
 
+  def show_camps
+    @camps = Camp.where(status: "active")
```

Comment:
Please make a scope of it and use that here i.e scope :active, -> { where(status: "active") }